src/index.ts

```diff
@@ -306,24 +311,23 @@ export class Client {
       filter.setEntityid(entityID)
       req.addFilters(filter)
     }
-    const client = grpc.client(API.Listen, {
+    const res = grpc.invoke(API.Listen, {
```

Comment:
Use the invoke API, rather that starting a new client. Technically, the above works, but it is more prone to open connections, and is really more suited to multiple unary calls than an single streaming response.